src/Dashboard/Provider.php

```diff
@@ -844,7 +843,6 @@ public static function nbItemByFk(
                 ],
                 'GROUPBY'   => "$fk_table.$name",
                 'ORDERBY'   => "cpt DESC",
-                'LIMIT'     => $params['limit'],
```

Comment:
You shouldn't just remove the limit, as it can be defined in the parameters passed to `nbItemByFk()`. Initially, when the limit wasn't defined, it defaulted to 50. Now, we still need to be able to set a limit, but if no parameter is specified, there should be no limit.